core/src/main/scala/org/apache/spark/status/storeTypes.scala

```diff
@@ -233,6 +243,38 @@ private[spark] class TaskDataWrapper(
     val shuffleLocalBytesRead: Long,
     @KVIndexParam(value = TaskIndexNames.SHUFFLE_READ_RECORDS, parent = TaskIndexNames.STAGE)
     val shuffleRecordsRead: Long,
+    @KVIndexParam(
```

Comment:
QQ: Do we have the requirement to sort task data by the new metrics?

Reply:
Although we do not sort these in the UI, adding indices for these are a requirement for computing quantiles and while fetching `TaskDataWrapper`s.